tests/test_runner.py

```diff
@@ -151,3 +151,55 @@ def test_runner_run_invalidated():
         assert any('Development switches or skip_system_checks were active for this run.' in msg for msg in messages)
     else:
         assert 'Development switches or skip_system_checks were active for this run. This will likely produce skewed measurement data.\n' in messages
+
+def test_runner_with_glob_pattern_filename():
+    """Test that runner works with glob pattern filenames like folder/*.yml"""
+    # Test runner.py with glob pattern that matches multiple files in a folder
+    ps = subprocess.run(
+        ['python3', 'runner.py', '--uri', GMT_DIR, '--filename', 'tests/data/usage_scenarios/runner_filename/*.yml',
+         '--skip-system-checks', '--dev-cache-build', '--dev-no-sleeps', '--dev-no-save'],
+        cwd=GMT_DIR,
+        capture_output=True,
+        text=True,
+        check=False
+    )
+
+    assert ps.returncode == 0, f"Runner failed with stderr: {ps.stderr}"
+    assert 'tests/data/usage_scenarios/runner_filename/basic_stress_1.yml' in ps.stdout
```

Comment:
this can easily flake, as it might also be part of a skip message. Please use matching that guarantees that it signals to be used in a run. If necessary patch scenario_runner to output this to stdout more clearly

Reply:
applies for line below too